src/main/java/edu/cornell/kfs/concur/batch/businessobject/ConcurStandardAccountingExtractDetailLine.java

```diff
@@ -18,6 +18,7 @@
     private String employeeStatus; 
     private String paymentCode; 
     private String journalAccountCode; 
+    private Boolean journalAccountCodeOverriden;
```

Comment:
I would suggest changing the name to use "...Overridden" instead of "...Overriden" (same with the getter and setter).